server/webhookNotifier.ts

```diff
@@ -250,14 +247,39 @@ const send = async (models, content: WebhookContent) => {
             },
           }]
         };
-      } else if (url.indexOf('matrix') !== -1) {
-        // TODO: matrix format and URL pattern matcher unimplemented
-        // return {
-        //   'text': `${getFiltered(content, address).join('\n')}`,
-        //   'format': 'plain',
-        //   'displayName': 'Commonwealth',
-        //   'avatarUrl': 'http://commonwealthLogoGoesHere'
-        // };
+      } else if (url.indexOf('telegram.org') !== -1) {
+        const getUpdatesUrl = url.split('/@').slice(0, -1).join('@');
+
+        var getChatUsername = url.split("/@");
```

Comment:
Use `let` instead of `var` here, and single-quotes rather than double-quotes throughout.